src/otx/algo/instance_segmentation/utils/utils.py

```diff
@@ -127,3 +132,276 @@ def empty_instances(
             results.masks = im_mask
         results_list.append(results)
     return results_list
+
+
+class MLP(nn.Module):
+    """Very simple multi-layer perceptron (also called FFN)."""
+
+    def __init__(self, input_dim: int, hidden_dim: int, output_dim: int, num_layers: int) -> None:
+        super().__init__()
+        self.num_layers = num_layers
+        h = [hidden_dim] * (num_layers - 1)
+        self.layers = nn.ModuleList(nn.Linear(n, k) for n, k in zip([input_dim, *h], [*h, output_dim], strict=True))
+
+    def forward(self, x: Tensor) -> Tensor:
+        """Forward pass."""
+        for i, layer in enumerate(self.layers):
+            x = f.relu(layer(x)) if i < self.num_layers - 1 else layer(x)
+        return x
+
+
+def inverse_sigmoid(x: Tensor, eps: float = 1e-5) -> Tensor:
```

Comment:
Same function exists at algo/detection/utils.py. How about moving a code to common space ans using it?

Reply:
Updated